draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -770,6 +770,8 @@ key-distribution method.
 
 # Privacy Considerations {#privacy_considerations}
 
+The privacy principles of [@ISO.29100] MUST be adhered to. 
```

Comment:
I don't feel it's appropriate to have a normative MUST pointing to a document that isn't freely available.

Reply:
It is actually publicly available - see url and search for the number

Reply:
My mistake. Sorry. I guess I'm accustomed to ISO docs being difficult to access.  Even so, I don't think a normative MUST to a different document is okay.

Reply:
I agree must is too strong. Would should be acceptable?

```suggestion
The privacy principles of [@ISO.29100] should be adhered to. 
```

Reply:
little should would be better, yes.

Reply:
github is telling me this conversation is outdated though so I can't commit the suggestion